specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2024-03-02-preview/managedClusters.json

```diff
@@ -3833,7 +3833,7 @@
         "tags": [
           "LoadBalancers"
         ],
-        "operationId": "LoadBalancers_Rebalance",
+        "operationId": "LoadBalancers_RebalanceLoadBalancers",
```

Comment:
As `LoadBalancers_RebalanceLoadBalancers` is regarded as a bad case, what do you think about changing the url?

Url: managedClusters/{resourceName}/loadBalancers/rebalance
OperationId: LoadBalancers_Rebalance

 https://github.com/Azure/azure-openapi-validator/blob/main/docs/operation-id-noun-verb.md

Reply:
fixed

Reply:
ok, got another error `Path for 'post' method on a resource type MUST follow valid resource naming.`
- https://github.com/Azure/azure-rest-api-specs/pull/28785/checks?check_run_id=24133521319
- https://github.com/Azure/azure-openapi-validator/blob/main/docs/path-for-resource-action.md

emmm.. is there something similar defined in the spec of nrp?

Reply:
> Url: managedClusters/{resourceName}/loadBalancers/rebalance

I dont think we should do this as it's not valid as per ARMs guidelines. What you had before was more correct: `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/rebalanceLoadBalancers`

Reply:
@matthchr yes I agree. But in that case there will be another alert complaining the operationId. Let me change it back.